### DISCUSSION TITLE
Automatically shrink neighbors?

### DISCUSSION THREAD

**Discussion by FezVrasta** [react-grid-layout/react-grid-layout#1424] [Q&A]
Hi, I was wondering if there's any way to automatically shrink neighbors cells when the user enlarges one?

I'm trying to make sure all the gaps are filled but it seems to be very challenging with the default behavior.

**Comment by STRML** [comment]
You'd have to modify the layout yourself to accomplish this after the resize. There's no automatic feature.

**Reply by FezVrasta** [reply]
Thanks, that means the other tiles will resize only after the resize operation is done though?

**Reply by STRML** [reply]
Yes, generally. You could listen to the drag/resize handlers if you want some other visual effect during the drag or resize, but it could get complicated, fast.
